docs/csharp/whats-new/csharp-10.md

```diff
@@ -22,7 +22,11 @@ C# 10 adds the following features and enhancements to the C# language:
 - [CallerArgumentExpression attribute](#callerargumentexpression-attribute-diagnostics)
 - [Enhanced `#line` pragma](#enhanced-line-pragma)
 
-Some of the features you can try are available only when you set your language version to "preview". These features may have more refinements in future previews before .NET 6 is released.
+In addition, there are two features that are available in *preview* mode. You're encouraged to try these features and provide feedback on them. They may change before their final release. In order to use these features, you must [set the `<LangVersion>` to `Preview`](../language-reference/compiler-options/language.md#langversion) in your project.
+
+The features available only in preview mode are:
+
+- [Generic attributes](#generic-attributes)
```

Comment:
Is something missing here? Perhaps CallerArgumentExpression?

Reply:
static abstract interfaces are coming in my next PR. I'll reword this for now.